### ISSUE TITLE
reporting mean values than the median from Bayesian mixed model using tab_model

### ISSUE THREAD

**Issue by preshitambade** [strengejacke/sjPlot#814]
My question was about extracting mean point estimates than the default 
the median in tab_model command for Bayesian models.

**Comment by preshitambade** [comment]
The documentation for tab_model has the function "bpe" which allows exporting mean values instead of median.

**Comment by strengejacke** [comment]
yes, the argument name is not very intuitive. I'm planning a general overhaul, including a more straightforward naming of arguments.

**Comment by preshitambade** [comment]
Thanks, that will be great. Also, please see if we can report Rhat values which denote model fit. [See 8.4.2.1 Convergence diagnostics.](https://bookdown.org/ajkurz/Statistical_Rethinking_recoded/markov-chain-monte-carlo.html) As you know we want Rhat=1 for convergence. Not sure if we can highlight these values when they are closer to one like p-value functionality. Thank you in advance.
